### PR TITLE
Provides additional information on Antivirus

### LINKED ISSUE #64: Update: Documentation around Antivirus changes for 1.7.x
Three current locations:

* https://www.archivematica.org/en/docs/archivematica-1.6/user-manual/troubleshooting/error-handling/
**NB** This page may need the image updating, but the rest of the text remains accurate.

* https://www.archivematica.org/en/docs/archivematica-1.6/getting-started/troubleshooting/faq/
**NB** This page remains accurate.

* https://www.archivematica.org/en/docs/archivematica-1.6/user-manual/transfer/transfer/
**NB** This page remains accurate.

Suggested changes: 

* Change between 1.6 and 1.7 i.e. no PREMIS event where virus scanning ostensibly didn't occur
* Where it is configured, i.e. via Environment variables for 1.7.x
* What our recommendation is, i.e. our defaults? Viruses should be in smaller objects
* What the impact of various configuration options? i.e. larger settings, longer processing times

### PR DESCRIPTION
**Pull request by ross-spencer**
This commit provides extra aministration information surrounding Antivirus
(AV) inside Archivematica. The information is linked to from the Admin and
user-manual transfer sections. We also update the error-handling pages to
make screenshots and process documentation up-to-date where AV was previously
referenced. We now focus on a checksum validation failure.

Resolves #64

### CODE REVIEW

**Review by sevein — COMMENTED** [review]
@ross-spencer please take a look at [this example](https://raw.githubusercontent.com/artefactual/archivematica-docs/0851b8cb0a479171162e478c4c1c177c73992d4c/admin-manual/security/security.rst) where the links are listed at the very bottom of the file. I think it's called internal cross-reference. The main benefit is that it lets you stick with the 80col convention.

**Comment by ross-spencer** [comment]
@sevein nice!

File: admin-manual/installation/customization/antivirus-admin.rst [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+	It is recommended that you employ Clamdscan for virus checking. 
+
+	Clamscan is slower than Clamdscan because the process has to be started 
+	and the virus database reloaded for each file that is passed to it. 
+
+	Clamdscan's utility is due in part to the antivirus definition database 
+	only being loaded once when the service is started. Files are passed to the 
+	service by stream or by filesystem reference.
+
+	To illustrate the difference in performace, a file that can on average take
```
Comment (sallain) [inline admin-manual/installation/customization/antivirus-admin.rst]: Suggested replacement for this paragraph:

To illustrate the difference in performance, a <number> MB <type (i.e. image)> file can be scanned by Clamdscan in an average of 2 seconds. Using Clamscan the average for the same file will be 1 minute 20 seconds.

File: admin-manual/installation/customization/antivirus-admin.rst [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+or not. If either of these are exceeded then Archivematica cannot guarantee 
+that a file has been scanned in its entirety for viruses. 
+
+Given that Archivematica cannot guarantee in such cases that all instances of 
+any virus have been detected a :ref:`PREMIS <premis-template>` event will not 
+be recorded.
+ 
+::
+
+	From the ClamAV manual pages:
```
Comment (sallain) [inline admin-manual/installation/customization/antivirus-admin.rst]: Would be nice to include a direct link, if possible.

File: user-manual/transfer/scan-for-viruses.rst [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+``MaxFileSize`` If a file is passed to the scanner that is larger than this 
+then it will not be scanned. No event will be recorded.
+
+``MaxScanSize`` limits the number of bytes that will be scanned. This might be 
+used in a standard operating environment where one might be confident a virus 
+or malware will only appear within that range. In Archivematica, because of the 
+possibility of a virus still existing outside of that range, a PREMIS event 
+cannot be recorded confidently.
+
+``MaxStreamLength`` is a setting used in Clamdscan only. the maximum number of
```
Comment (sallain) [inline user-manual/transfer/scan-for-viruses.rst]: Missing a capital T :)

**Review by sallain — APPROVED** [review]
A few small change suggestions, but otherwise this looks good to go!

File: admin-manual/installation/customization/antivirus-admin.rst (hunk 2 above)
Comment (ross-spencer) [reply admin-manual/installation/customization/antivirus-admin.rst]: I found a nice version of the docs at Debian, if that seems okay: https://manpages.debian.org/jessie/clamav-daemon/clamd.conf.5.en.html

**Comment by ross-spencer** [comment]
@sallain changes made! Just need a thumbs up if okay to merge :)

**Review by sallain — APPROVED** [review]

**Pull request merged**